core/src/scene/drawRule.h

```diff
@@ -37,11 +38,10 @@ class StyleContext;
 struct DrawRuleData {
 
     std::vector<StyleParam> parameters;
-    std::string name;
-    int id;
+    std::string styleName;
+    int styleId;
```

Comment:
Ok, I see, DrawRule.name for the rule name makes sense. reverting and adding a comment for my future self.. :)